src/components/main/sidebar/mod.rs

```diff
@@ -51,7 +51,11 @@ pub fn Sidebar(cx: Scope<Props>) -> Element {
     let notifications_tx = use_coroutine(&cx, |mut rx: UnboundedReceiver<Message>| async move {
         while let Some(msg) = rx.next().await {
             let display_username = utils::get_username_from_did(msg.sender().clone(), &mp);
-            PushNotification(display_username, msg.value().join("\n"));
+            PushNotification(
+                display_username,
+                msg.value().join("\n"),
+                "Usual Notification".to_owned(),
```

Comment:
`notification_sound` should be an `enum`, not a string. looks like `sounds::Sounds` already provides such an `enum`

Reply:
You're correct, I've changed it.